### ISSUE TITLE
MacOS podman images fails 

### ISSUE THREAD

**Issue by ashley-cui** [containers/podman#7610]
/kind bug

**Description**

podman images on mac fails 
**Steps to reproduce the issue:**

$ podman images 
Error: parsing time "1581706310" as ""2006-01-02T15:04:05Z07:00"": cannot parse "1581706310" as """

**Describe the results you expected:**
same as podman images on local client

**Additional information you deem important (e.g. issue happens only occasionally):**
podman-remote on linux doesn't do this

**Output of `podman version`:**
podman version 2.0.6

**Comment by zhangguanzhang** [comment]
I think could use dlv to debug this, but I work on windows, I can't help

**Comment by afbjorklund** [comment]
There are some extra quotes in there, which is causing this error.

```go
package main

import (
	"fmt"
	"log"
	"time"
)

func main() {
	t, err := time.Parse("\"2006-01-02T15:04:05Z07:00\"", "1581706310")
	if err != nil {
		log.Fatal(err)
	}
	fmt.Println(t)
}

```

2009/11/10 23:00:00 parsing time "1581706310" as ""2006-01-02T15:04:05Z07:00"": cannot parse "1581706310" as """

**Comment by afbjorklund** [comment]
Possibly https://github.com/golang/go/issues/23134

**Comment by ashley-cui** [comment]
huh, an issue in 2.0.6 but solved on master. welp, another issue to close

**Comment by rhatdan** [comment]
Should be fixed in podman 2.1 which is in RC1 at this point.
